x/oracle/ibc_test.go

```diff
@@ -195,19 +166,29 @@ func (suite *OracleTestSuite) TestIBCPrepareRequestNotEnoughFund() {
 		1,
 		1,
 		sdk.NewCoins(sdk.NewCoin("uband", sdk.NewInt(3000000))),
-		"beeb-request",
 		testapp.TestDefaultPrepareGas,
 		testapp.TestDefaultExecuteGas,
 	)
+
+	// Use Carol as a relayer
```

Comment:
Let's send more fund to this relayer to make sure he can pay on some data sources but not enough to pay all data sources and check the balance of treasury and carol still the same.